src/6_branch.js

```diff
@@ -67,6 +67,10 @@ var wrap = function(parameters, func, init) {
 			 * @type {function(?Error,?): undefined}
 			 */
 			var done = function(err, data) {
+				if (err) {
```

Comment:
Line 74 is still useful, I think. The problem was that if there was an error but no callback, the error was being thrown, which broke the callback chain in the task queue since nothing was calling next().  This meant that the next task added to the queue would never be fired, since the first task wasn't actually removed.